client/lib/application/src/application-lifecycle/components/Loading.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ThreeColumns from 'common/src/components/pure/ThreeColumns.jsx';
+import Icon from 'react-fa';
+
+const Loading = () => {
+    return (
+        <ThreeColumns
+            leftChildren   = {<div></div>}
+            middleChildren = {<div><Icon pulse size='5x' name='spinner' /> Loading</div>}
```

Comment:
will change to circle-o-notch.
as for pulse in conjunction with spinner see:
http://fontawesome.io/examples/ in "animated icons" last icon :)